.vale/styles/Vocab/Train/reject.txt

```diff
@@ -0,0 +1 @@
+HuggingFace
```

Comment:
I'd recommend adding this to `WordList.yml` instead.

If you use `reject.txt`, you get this error message:

> Avoid using 'HuggingFace'.

In contrast, if you use `WordList.yml`, you get this error message:

> Use 'Hugging Face' instead of 'HuggingFace'.

Reply:
ack - I've committed those changes. Could you take a look? @bveeramani